gfx/src/sdl2/command_buffer_impl.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright Antioch. All rights reserved.
+#include <gfx/gfx.h>
+
+#include "base/function_impl.h"
+#include "common/allocation.h"
+#include "sdl2/objects.h"
+
+namespace antioch::gfx::IMPL {
+CommandBuffer_t* implAllocateCommandBuffer(const AllocationCallback* pAllocator) {
```

Comment:
I see a lot of C and not a lot of C++ here

Reply:
hol up I got a cook